README.md

```diff
@@ -21,36 +21,81 @@ Why is it important that we use typing in our database? Simply put, typing allow
 
 Typing gives us the ability to perform all kinds of operations with predictable results. For instance, the ability to perform Math operations like SUM - i.e. summing integers - doesn't just depend on everything being an integer of some sort but would also expect it. If you tried, for example,  to SUM all of the dogs in the above table, SQLite would actually attempt to convert, or cast, their type to something it can SUM. It would try to convert anything it can to an INTEGER and ignore alpha characters. This can lead to real problems. Without typing, our data might get complicated and messy, and it would be difficult to ask the database questions about large sets of data.
 
 We're going to adhere strictly to only storing data that fits with the datatype we have given to a particular column.
 
-## Data Types
+## Datatypes
 
-Different database systems also have different datatypes available, which are important and useful to know whenever you are dealing with those systems. In SQLite there are five different types we will be dealing with, they are:
+Different database systems also have different datatypes available, which are important and useful to know whenever you are dealing with those systems. In SQLite there are five basic categories of datatypes we will be dealing with, they are:
 
 ```bash
 TEXT
 NUMERIC
 INTEGER
 REAL
-NONE
+BLOB
 ```
 
-For now we'll cover just three types, and later on we'll also learn about a couple other useful types not listed here. Our types:
+These categories each contain several datatypes. Let's explore each category in more detail.
 
 ### TEXT
 
 Any alphanumeric characters which we want to represent as plain text. The body of this paragraph is text. Your name is text. Your email address is a piece of text. Your height, weight, and age, however, are probably not.
 
+Common text datatypes include:
+
+```bash
+	CHARACTER(20)
+	VARCHAR(225)
+	TEXT
+```
+
 ### INTEGER
 
 Anything we want to represent as a whole number. If it's a number and contains no letter or special characters or decimal points then we should store it as an integer. If we use it to perform math or create a comparison between two different rows in our database, then we definitely want to store it as an integer. If it's just numbers, it's generally not a bad idea to store it as an integer. You might never add two house address numbers together, but you might want to sort them numerically. For example in the preceding case, you might want to get the biggest number and not the longest piece of text.
 
+Common integer datatypes include:
+
+```bash
+	INT
```

Comment:
One liner explanation of the diff, and then have an opinion. Which one should we use? Whenever we give options, we should give opinions